### ISSUE TITLE
admin-ajax  add_post_meta don't translate for not default idiome in the 2.0

### ISSUE THREAD

**Issue by pulluptech** [maximeschoeni/sublanguage#20]
$ids = array(1,2,3,4,5);
add_action( 'wp_ajax_import_product', 'import_product' );
$key = "_attribs_ids";
add_post_meta($post_id, $key, $ids);

global $sublanguage; // return null

**Comment by maximeschoeni** [comment]
Please can you verify if the "Use AJAX in Front-End" checkbox is checked in wp_admin > Settings > Sublanguage?

**Comment by pulluptech** [comment]
I'ts enable...

I found the problem, when i click in sub tab idiome the plugin don't copy the meta values to new one for transalate then, the get_post_meta return a empty array.

**Comment by maximeschoeni** [comment]
And is your meta_key set to be translatable in the "language options" page of the corresponding post_type?

**Comment by pulluptech** [comment]
Yes, the meta it's an array

**Comment by maximeschoeni** [comment]
Right, there was actually a bug when the meta was an array. I think I fixed it. Would you like to download and try the newer version (from github) and confirm me its fixed?
